### PR TITLE
Update latest_news.html

### PR DESCRIPTION
**Pull request by AshtonDavis**
1.7 is out, and a new newsletter has been released.  I updated both links and descriptions.

### CODE REVIEW

File: _includes/latest_news.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<p>Foreman 1.6.0 has been released, with APIv2, Smart Proxy plugin support, external/LDAP group support and more new features. <a href="/manuals/1.6/index.html#Releasenotesfor1.6">Read the release notes</a>.</p>
+<p>Foreman 1.7.0 has been released.  Highlights include bonded and virtual networking support, APIv2 enhancements, Puppet Smart Class enhancements, build review, and more. <a href="/manuals/1.7/index.html#Releasenotesfor1.7">Read the release notes here</a>.</p>
 <hr />
-<p>Foreman Community Newsletter - May 2014: the release of Foreman 1.5, exciting OpenNebula developments and various Foreman-related articles. <a href="http://blog.theforeman.org/2014/05/foreman-community-newsletter-may-2014.html">Read the newsletter</a>.</p>
+<p>Foreman Community Newsletter - May 2014: the release of Foreman 1.7, Discovery 2.0, orchestration, and various other Foreman-related articles. <a href="http://blog.theforeman.org/2014/12/foreman-community-newsletter-november.html">Read the newsletter</a>.</p>
```
Comment (GregSutcliffe) [inline _includes/latest_news.html]: November 2014, not May 2014 :)

Comment (GregSutcliffe) [inline _includes/latest_news.html]: Also it's Discovery Image 2.0 specifically, the plugin hasn't hit 2.0 yet

Comment (AshtonDavis) [inline _includes/latest_news.html]: Well fine, I guess I'll fix it then. :(

**Comment by AshtonDavis** [comment]
Hey Greg, I fixed it.  I've got the hang of commit squashing by now, I think.

**Comment by domcleal** [comment]
Thanks @ntent-ashton!

**Pull request merged**